src/FirePoke.java

```diff
@@ -0,0 +1,75 @@
+package src;
+
+public class FirePoke extends Pokemon {
+
+    //VARIABELEN /////////////////////////
+    private String heat;
+    private String heatDamage;
+
+
+    //CONSTRUCTOR
+    FirePoke(String type, String name, int height, int weight, int damage, String sound, String food, String heat, String heatDamage) {
+        super(type,name, height, weight, damage, sound, food);
+
+        this.heatDamage = heatDamage;
+        this.heat = heat;
+
+
+    }
+
+    //METHODS //////////////////////////
+
+    @Override
+    public void eats(String food) {
+
+        System.out.println(getName() + " eats: " + getFood());
+    }
```

Comment:
2x zelfde override in Electric en FirePokemon. Dan kan je beter deze sout meegeven in je superClass